### DISCUSSION TITLE
Quarto website for rainbowR

### DISCUSSION THREAD

**Discussion by EllaKaye** [quarto-dev/quarto-cli#11651] [Show and Tell]
### Description

Prompted by @Lextuga007, I'm sharing the rainbowR website in this 'Show and Tell':

site: <https://rainbowr.org>
code: <https://github.com/r-lgbtq/rainbowr.org>

My favourite feature is the way that the navbar menu items, when hovered over, turn different (rainbow) colours, and an animated line of the same colour appears underneath. (This was a feature I first implemented in my [personal site](https://ellakaye.co.uk) then co-opted for rainbowR).

There are also two custom syntax highlighting schemes (one for light mode and one for dark mode), which tone well with the rainbow colour palette whilst ensuring that WCAG colour-contrast guidelines are met. The section on the [implementation of the buddies scheme](https://rainbowr.org/buddies.html#implementation) is a good place to see these in action.

I also make use of my outdated-but-still-functioning [distilltools](https://github.com/EllaKaye/distilltools/) package to create what appear to be nicely formatted buttons with an icon. The [`icon_link(icon, text, url)`](https://ellakaye.github.io/distilltools/reference/icon_link.html) function produces html with an `icon-link` class, that can then be styled through css. See, for example, the 'register' link on the [meet-ups](https://rainbowr.org/meetups.html) page. (A more Quarto-native way to do this would probably be through the `btn` class and using a shortcode/extension for the icon, but I went with what I knew at the time!)

@cderv, would you consider adding this site to the websites section of the Quarto gallery? I think there's enough about it that's different to what's already there, plus it would be great visibility for the rainbowR community 🏳️‍🌈🏳️‍⚧️.

**Comment by cderv** [comment]
> would you consider adding this site to the websites section of the Quarto gallery? I think there's enough about it that's different to what's already there, plus it would be great visibility for the rainbowR community 🏳️‍🌈🏳️‍⚧️.

yes definitely. You can open a PR there, or I'll find time to do it.

And nice website !! 🎉

**Reply by mcanouil** [reply]
@EllaKaye This would also be a good addition to https://github.com/mcanouil/awesome-quarto?tab=readme-ov-file

**Comment by EllaKaye** [comment]
@cderv I thought you might like to know that we just had someone join the rainbowR community who found out about us after seeing our website on the Quarto Gallery. Thank you so much for including it there 🌈

**Reply by cderv** [reply]
Wow! So cool ! Thanks for sharing !
